source/layouts/layout.slim

```diff
@@ -57,3 +57,6 @@ html
         .footer-up
           a.caret-up href="#"
             img.caret-up-img src="/assets/components/footer/icon-arrow-up.svg" /
+
+  javascript:
```

Comment:
I usually don't do it on Analytics since it might make hits get lost if the page is loading slowly, but this loads quickly enough that we shouldn't worry about it 😁